cfgmgr/intfmgr.cpp

```diff
@@ -428,16 +428,15 @@ bool IntfMgr::doIntfGeneralTask(const vector<string>& keys,
 
     string alias(keys[0]);
     string vlanId;
-    string subIntfAlias;
+    string parentAlias;
```

Comment:
Why this change? Its not matching the PR description

Reply:
It is the change in https://github.com/Azure/sonic-swss/pull/1521. This PR only contains change in vs test, developed on top of https://github.com/Azure/sonic-swss/pull/1521. However, without https://github.com/Azure/sonic-swss/pull/1521, vs test will fail